src/document/DocumentCommandHandlers.js

```diff
@@ -1598,6 +1602,55 @@ define(function (require, exports, module) {
         ProjectManager.showInTree(MainViewManager.getCurrentlyViewedFile(MainViewManager.ACTIVE_PANE));
     }
 
+    /** 
+     * Show error dialog
+     * @param {string} err Error message to be displayed to the user
+     * @param {callback} callback
+     */
+    function showErrorDialog(err, callback){
+        callback = callback || function () {};
```

Comment:
Do we really need to support the case of having no callback?  Aren't you the only caller of this?